plugins/cloudimport/templates/load_buttons.js

```diff
@@ -12,9 +12,17 @@ PluginsAPI.Dashboard.addNewTaskButton(
 PluginsAPI.Dashboard.addTaskActionButton(
 	["cloudimport/build/TaskView.js"],
 	function(args, ImportView) {
-        return React.createElement(ImportView, {
-                task: args.task,
-                apiURL: "{{ api_url }}",
-        });
+		$.ajax({
+			url: `{{ api_url }}/projects/${args.task.project}/tasks/${args.task.id}/checkforurl`,
```

Comment:
Careful using ES6 features (string templates, for example) in non .jsx files (which are not going to be built with webpack). The code will not work on older browsers.

Reply:
Removed the string tempaltes 😄